### DISCUSSION TITLE
Branching and edge functions

### DISCUSSION THREAD

**Discussion by novemberkilo** [supabase/supabase#28987] [Questions] [answered]
Hello - I've created a preview branch and seen migrations run successfully. I'm expecting to see the edge functions (that are on production) to appear on the preview branch as well but they don't. Is this normal behaviour and if so, what's the recommended workflow for deploying edge functions to the branch? Similarly, if I create a new edge function on my preview branch, should I expect that to appear on production when I merge preview? 

Thank you.

**Comment by gc-ft** [comment]
Same issues here! @MildTomato could you please comment on this problem?

The comments created by the GitHub integration even MENTION Edge Functions and confirm their status. But in the branch on supabase the edge functions just do not appear!

**Reply by LewisFramps** [reply, marked as answer]
I can't find official documentation on this - but I did find this within https://github.com/supabase/supabase/blob/master/supabase/config.toml

```
[functions.search-embeddings]

verify_jwt = false
```

just adding 
`[functions.func-name-here]` to my config.toml fixed this issue and got them deploying on branches
